tests/js/setup-before-after.js

```diff
@@ -14,5 +16,5 @@ beforeEach( () => {
 		sessionStorage[ method ].mockClear();
 	} );
 
-	fetch.resetMocks();
+	fetchMock.reset();
```

Comment:
It seems like this belongs more in `afterEach` but I see all the others are here so I suppose that's okay. @tofumatt any reason why this is in before instead of after?

```suggestion
	fetchMock.mockReset();
```

Reply:
Yeah, afterEach() makes more sense. I've moved it to an `afterEach` block.